convert/src/vertex_to_bd/models/vertex_models.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel
+
+from .common import S3Path
+
+
+class VertexRequest(BaseModel):
+    input_annotation: S3Path
+    output_pdf: Optional[S3Path]
+    output_tokens: Optional[S3Path]
+    output_annotation: Optional[S3Path]
+
+
+class BadgerdocToVertexRequest(BaseModel):
+    input_tokens: S3Path
+    input_annotation: S3Path
+    output_annotation: S3Path
+
+
+class Value(BaseModel):
+    start: int
+    end: int
+    text: str
+    labels: List[str]
+
+
+class ResultItem(BaseModel):
+    value: Optional[Value] = None
+    id: str = None
```

Comment:
id cannot be None

Reply:
https://github.com/epam/badgerdoc/issues/170